src/charts/c3/c3.mixin.js

```diff
@@ -0,0 +1,57 @@
+angular.module('patternfly.charts').factory('ChartsMixin', function() {
```

Comment:
Sorry, the cfmePOC is a dashboard for containers where we initially developed a lot of the c3 charts (trends, utilization, donut, sparkline, etc.).  We are slowly moving these charts into a-pf#master.